src/components/DeleteTableDialog.vue

```diff
@@ -193,15 +193,23 @@ export default Vue.extend({
         workspace,
       } = this;
 
-      const graphNames = await api.graphs(workspace);
-
+      const graphNames = (await api.graphs(workspace)).results.map((graph) => graph.name);
       const using = [] as Array<{graph: string; tables: string[]}>;
       graphNames.forEach(async (graph) => {
-        const data = await api.graph(workspace, graph);
+        const nodes = await api.nodes(workspace, graph, {});
+        // eslint-disable-next-line no-underscore-dangle
+        const edges = await api.edges(workspace, graph, nodes.results[0]._id, {
+          direction: 'all',
+        });
 
+        // eslint-disable-next-line no-underscore-dangle
+        const prelimNodes = nodes.results.map((node) => node._id.substring(0, node._id.indexOf('/')));
```

Comment:
I think you can accomplish the splitting on `/` better with the use of [`String.split()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split) instead of the combination of `substring()` and `indexOf()`.

Reply:
See https://github.com/multinet-app/multinet-client/commit/ceedc898a945927e4e277d13583c9eec195e6c64#diff-59d96b3b10e2aff2a38e84b28bc951f257462a7772757d307d3952647372bd4dR206